curriculum/challenges/english/25-front-end-development/lab-profile-lookup/5688e62ea601b2482ff8422b.md

```diff
@@ -0,0 +1,154 @@
+---
+id: 5688e62ea601b2482ff8422b
+title: Build a Profile Lookup
+challengeType: 26
+dashedName: profile-lookup
+---
+
+# --description--
+
+In this lab, you will build a profile lookup that looks up information about people in a contacts list.
+
+For this example imagine there is a contact named Akira Laine, the `lookUpProfile("Akira","lastName")` should return `Laine`.  
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab
+
+**User Stories:**
+
+1. You should create a function named `lookUpProfile` that takes a name and a property as arguments.
+2. If the function receives a contact name and the property exists on that contact; the property's value should be returned.
+3. If the name passed to the function does not match any contacts, then the function should return `No such contact`.
+4. If the property does not exist on a found contact, then the function should return "No such property". 
+
+# --hints--
+
+You should have a function named `lookUpProfile`.
+
+```js
+assert.isFunction(lookUpProfile);
+```
+
+`lookUpProfile` should take two strings as arguments.
+
+```js
+assert.match(lookUpProfile.toString(), /\s*function\s+lookUpProfile\s*\(\s*\w+\s*\,\s*\w+\s*\)/);
+```
```

Comment:
you can't test with this that the function is taking two string as arguments. What you are testing is that the function is defined with two parameters here. I suggest removing the test, or replacing the hint

Reply:
I'm removing the test.